packages/uxpin-merge-cli/src/program/command/delete_version/steps/DeleteRepositoryPointer.ts

```diff
@@ -0,0 +1,94 @@
+import { DEFAULT_BRANCH_NAME } from '../../../../../src/common/constants';
+import { PrintColor } from '../../../../../src/utils/console/PrintOptions';
+import { deleteRepositoryPointerToBranch } from '../../../../common/services/UXPin/DeleteRepositoryPointerToBranch';
+import { deleteTag } from '../../../../common/services/UXPin/DeleteTag';
+import { getApiDomain } from '../../../../common/services/UXPin/getApiDomain';
+import { DSMetadata } from '../../../../program/DSMeta';
+import { DeleteOptions } from '../../../../steps/deleting/DeleteOptions';
+import { VCSDetails } from '../../../../steps/serialization/DesignSystemSnapshot';
+import { printError, printLine } from '../../../../utils/console/printLine';
+import { StepExecutor } from '../../Step';
+
+export function deleteRepositoryPointer(deleteOptions:DeleteOptions):StepExecutor {
+  return async (designSystem:DSMetadata) => {
+    const apiDomain:string = getApiDomain(deleteOptions.uxpinApiDomain!);
+    const authToken:string = deleteOptions.token!;
+    const vcsDetails:VCSDetails = designSystem.result.vcs;
+    const commitHash:string = vcsDetails.commitHash;
+    const branch:string | undefined = vcsDetails && vcsDetails.branchName !== DEFAULT_BRANCH_NAME
+    ? vcsDetails.branchName
+    : undefined;
+    const tag:string | undefined = deleteOptions.tag;
```

Comment:
when I specify --branch and --tag parameters only tag will be removed. Maybe we should add an additional message that only one parameter should be defined ?

Reply:
Sounds good! so for example if a user specifies --branch and --tag we should have message that tells them there only needs to be one option?

Reply:
yep :)